DOCS/contribute.md

```diff
@@ -131,12 +131,22 @@ Touching user-visible parts may require updating the mpv docs
 - Changes to command line options (addition/modification/removal) must be
   documented in options.rst.
 - Changes to input properties or input commands must be documented in input.rst.
-- All incompatible changes to the user interface (options, properties, commands)
-  must be documented with a small note in interface-changes.rst. (Additions may
-  be documented there as well, but this isn't required.)
 - Changes to the libmpv API must be reflected in the libmpv's headers doxygen,
   and in client-api-changes.rst.
 
+Interface change policy
+-----------------------
+
+- All incompatible changes to the user interface (options, properties, commands)
+  must be documented by making a new text file with a txt extension containing a
```

Comment:
> by making a new text file with a txt extension containing a

I think we should standardize file naming in this `interface-changes` to avoid silly naming. Probably something like commit msg when it were added is ok.

Reply:
Added a brief line after this. Not sure how much we want to dictate this atm. I'm personally fine as long as the name is clearly related to the actual change and I guess they don't put a bunch of weird characters in the name. We could always run a CI lint job on this if need be.

Reply:
I fine with anything, as long as I don't have to open the file to understand what change is it relate too.